.gitignore

```diff
@@ -3,4 +3,5 @@ node_modules
 .vscode-test/
 *.vsix
 
-demo.gif
\ No newline at end of file
+demo.gif
+tmp
```

Comment:
I use [pre-commit](https://pre-commit.com/) on most of my projects. It can be a helpful addition to cases like this. I'm sure `npm` has something similar.